### ISSUE TITLE
Consider using control sequences for terminal resizing on playback

### ISSUE THREAD

**Issue by spbnick** [Scribery/tlog#94]
Apparently XTerm, and some other terminal emulators support control sequences for resizing the terminal window. Found in asciinema/asciinema#198. Most of the terminals appear to require explicit enabling of this feature, as it can be abused.

However, we can provide that option in tlog-play, and insert these sequences into the output when reading the resize entries, so that users who wish to use XTerm, or gnome-terminal, and see resizes can do that.

**Comment by spbnick** [comment]
To allow window operations for a single xterm instance this command line can be used:

    uxterm -xrm 'xterm*allowWindowOps: true'

**Comment by spbnick** [comment]
Then a command line like this can be used to try to resize the xterm window:

    echo -ne '\e[8;60;120t'

This command resizes it to 120x60 characters.

**Comment by timofonic** [comment]
@sickill @dhobsd @EliteTK @marchelzo @XANi @mattdeckard @vvv @xloem @josegonzalez @guyromm @gizmonicus @XANi @oblique @blueyed @losingkeys @mrabault @sigmavirus24 @daGrevis @spbnick @elopio @cortezcristian @alex @AJNOURI @louisje @theonewolf @galdiolo @flyinbutrs @jmhobbs @chjj @zhiyelee @zyrolasting @FallingSnow @handmilkingsoftware @probonopd @dzan @ellerbrock  @theatrus @mneumann @nmbarker @posto @debben @cgag @regular @bastiaanb @IonicaBizau @dbkaplun @matthiasg @jfinstrom @jmazon @valtam @leo @chabou @zeit @albinekb @ppot @nesl247 @sheerun @Stanzilla @railsware @vlad-shatskyi @drew-gross @yudai @sourcelair @Tyriar @ConradIrwin @asharpe-squiz @somian @anapsix @kyokley @Scribery Any news about this?

**Comment by XANi** [comment]
@spbnick could you ban that guy ? he did same spam in asciinema project

**Comment by spbnick** [comment]
Yeah, I know, he mentioned me, but I wouldn't ban somebody for just being silly, unless they persist.

**Comment by sigmavirus24** [comment]
Y'all can use https://github.com/contact/report-abuse?report=timofonic+(user) to report their behaviour to github though.

**Comment by spbnick** [comment]
Thank you, Ian.
